bentoml/cli/deployment.py

```diff
@@ -502,18 +239,14 @@ def apply(deployment_yaml, output, wait):
         try:
             yatai_client = YataiClient()
             result = yatai_client.deployment.apply(deployment_yaml)
-            if result.status.status_code != status_pb2.Status.OK:
-                _echo(
-                    'Failed to apply deployment {name}. '
-                    '{error_code}:{error_message}'.format(
-                        name=deployment_yaml.get('name'),
-                        error_code=status_pb2.Status.Code.Name(
-                            result.status.status_code
-                        ),
-                        error_message=result.status.error_message,
-                    ),
-                    CLI_COLOR_ERROR,
-                )
+            error_code, error_message = parse_pb_response_error_message(result.status)
+            if error_code and error_message:
+                if result.status.status_code != status_pb2.Status.OK:
+                    _echo(
+                        f'Failed to apply deployment {deployment_yaml.get("name")}. '
+                        f'{error_code}:{error_message}',
+                        CLI_COLOR_ERROR,
+                    )
             else:
                 if wait:
```

Comment:
any reason this is implemented differently?

Reply:
lol, didnt update it